src/main/java/org/jitsi/videobridge/octo/OctoEndpoint.java

```diff
@@ -78,6 +78,25 @@ public void requestKeyframe(long mediaSsrc)
         }
     }
 
+    @Override
+    public void requestKeyframe()
+    {
+        // just making sure the tentacle hasn't expired
+        OctoTentacle tentacle = getConference().getTentacle();
+        if (tentacle != null)
+        {
+            long firstVideoSsrc = receiveSsrcs.entrySet().stream()
```

Comment:
Do we have the same problem with RTX SSRCs as in JMT?

Reply:
hm, i suspect we will.  will fix that.